src/timesync/fuzz-nts-extfields.c

```diff
@@ -0,0 +1,80 @@
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "fuzz.h"
+
+#include "nts.h"
+#include "nts_crypto.h"
+#include "nts_extfields.h"
+
+static void eat(volatile const uint8_t* buf, size_t size) {
+        if (buf) while (size) (void)buf[size--];
```

Comment:
this is not our coding style. please never stuff so much stuff in one line. this needs two line breaks in the middle.

also, this loop will be optimized away by any compiler, i don't get it?